var/spack/repos/builtin/packages/hpgmg/package.py

```diff
@@ -26,22 +26,24 @@
 
 
 class Hpgmg(Package):
-    """HPGMG implements full multigrid (FMG) algorithms using
-    finite-volume and finite-element methods.
-    Different algorithmic variants adjust the arithmetic intensity
-    and architectural properties that are tested. These FMG methods
-    converge up to discretization error in one F-cycle,
-    thus may be considered direct solvers. An F-cycle visits
-    the finest level a total of two times,
-    the first coarsening (8x smaller) 4 times,
-    the second coarsening 6 times, etc."""
+    """HPGMG implements full multigrid (FMG) algorithms using finite-volume and
+       finite-element methods.  Different algorithmic variants adjust the
+       arithmetic intensity and architectural properties that are tested. These
+       FMG methods converge up to discretization error in one F-cycle, thus may
+       be considered direct solvers. An F-cycle visits the finest level a total
+       of two times, the first coarsening (8x smaller) 4 times, the second
+       coarsening 6 times, etc.
+    """
 
     homepage = "https://bitbucket.org/hpgmg/hpgmg"
     url      = "https://hpgmg.org/static/hpgmg-0.tar.gz"
     tags     = ['proxy-app']
 
     version('a0a5510df23b', 'b9c50f25e541428d4735fb07344d1d0ed9fc821bdde918d8e0defa78c0d9b4f9')
     version('develop', git='https://bitbucket.org/hpgmg/hpgmg.git', branch='master')
+    version('a0a5510', git='https://bitbucket.org/hpgmg/hpgmg.git',
+            commit='a0a5510df23b0f8ab0decf5d084175adb6418c81')
```

Comment:
@jedbrown, it looks like the same hash is used to define two separate versions: `a0a5510df23b` and `a0a5510`.

Is this OK with you?

Reply:
That hash denotes the same version. In the top line, the sha256 is of the tarball. In the bottom line, it says ot clone from Git. Probably only one of them should remain.

Reply:
Do you want to make the change before we merge this PR?

Reply:
Sure, I assume that sticking with the tarball is okay. The git clone is a bit slow due to some heavy documentation.